services/common/src/components/tailings/BasicInformation.tsx

```diff
@@ -133,94 +132,104 @@ export const BasicInformation: FC<BasicInformationProps> = (props) => {
       <Field
         id="facility_type"
         name="facility_type"
-        label="Facility Type *"
+        label="Facility Type"
         component={renderConfig.SELECT}
         disabled={!canEditTSFAndEditMode}
         data={FACILITY_TYPES}
-        validate={[requiredList, validateSelectOptions(FACILITY_TYPES)]}
+        required
+        validate={[requiredList]}
```

Comment:
I know this wasn't working quite properly, but as the intention of this validation was to ensure the selection was one of a valid list of options did you look into a new version that might substitute?

Reply:
It's actually completely unnecessary, as far as I can tell! I think it was basically a bug with old antd. I cannot select an option that isn't one of the options, or submit with the search text still in the select.